emails/2023-01-13.md

```diff
@@ -0,0 +1,41 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+I'm in Telluride, Colorado all week on a family ski trip. You're still getting the email because I'm dedicated. As I get older, my goal for ski trips is to just not end of like this:
```

Comment:
```suggestion
I'm in Telluride, Colorado all week on a family ski trip. You're still getting the email because I'm dedicated. As I get older, my goal for ski trips is to just not end up like this:
```